style.css

```diff
@@ -0,0 +1,382 @@
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+
+* {
+  font-family: 'COCOGOOSE', sans-serif;
+}
+
+body {
+  width: 100%;
+}
+
+.main-nav,
+.navigation,
+.link-to-program,
+.last-text {
+  display: none;
+}
+
+#first-header {
+  background-color: rgba(227, 224, 224, 0.9);
+  height: 10vh;
+}
+
+.top-bg {
+  background-image: url("assets/backgroundImg.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#menu-image {
+  font-size: 30px;
+}
+
+.hero {
+  color: rgb(255, 64, 0);
+}
+
```

Comment:
- [ ] As suggested already, I think your title text color could be improved on, the color used in the design is `#EC5242`